lib/cc/services/github_pull_requests.rb

```diff
@@ -87,18 +81,18 @@ def update_status_failure
   end
 
   def presenter
-    CC::Service::GitHubPullRequestsPresenter.new(@payload)
+    CC::Service::PullRequestsPresenter.new(@payload)
   end
 
   def update_status_error
     update_status(
       "error",
-      @payload["message"] || DEFAULT_ERROR
+      @payload["message"] || presenter.error_message
     )
   end
 
   def update_status_pending
-    update_status("pending", "Code Climate is analyzing this code.")
+    update_status("pending", presenter.pending_message)
```

Comment:
There's no harm in adding the `@payload["message"] ||` part here now right? I'm going to need it soon.